sdk/storagepool/Azure.ResourceManager.StoragePool/Azure.ResourceManager.StoragePool.sln

```diff
@@ -9,6 +9,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.ResourceManager.Stora
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.Core.TestFramework", "..\..\core\Azure.Core.TestFramework\src\Azure.Core.TestFramework.csproj", "{77EE59F4-F06D-4FAE-974B-F6DF9B566575}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.ResourceManager", "..\..\resourcemanager\Azure.ResourceManager\src\Azure.ResourceManager.csproj", "{4396D0FA-8719-40B1-8A60-0039C6EEF11F}"
```

Comment:
Is there any reason that we add the project `Azure.ResourceManager` here?

Reply:
Removed